src/medication-dispense/medication-dispense.resource.tsx

```diff
@@ -39,16 +39,20 @@ export function initiateMedicationDispenseBody(
     let dispense = {
       resourceType: "MedicationDispense",
       status: "on-hold", // might need to change this to appropriate status
+      authorizingPrescription: {
+        reference: "MedicationRequest/" + medicationRequest.id,
+        type: "MedicationRequest",
+      },
```

Comment:
@pirupius  @ibacher  just interested if manually building the reference this way is "correct"... in this case, we have pulled down the entire medication request, not the the reference to it, so we didn't seem to have the reference version available.

Reply:
Yeah, I think it's fine to do. You don't necessarily need the `type`, though it's probably harmless.